### PR TITLE
Create feature layout heaader website

### PR DESCRIPTION
**Pull request by nhatanh19**

### CODE REVIEW

File: src/stylesheets/components/_button.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    background-color: $color-main;
+    border-radius: 50px;
+    border-color: $color-main;
+
+    &:hover {
+        cursor: pointer;
+        transform: scale(1.1);
+    }
+
+    .btn__content {
```
Comment (Hoa0304) [inline src/stylesheets/components/_button.scss]: BEM in SCSS?

File: src/stylesheets/layouts/_header.scss [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+    .header-top {
+        width: 100%;
+        height: pxToRem(95);
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding: 0 pxToRem(120);
+
+        // logo
```
Comment (Hoa0304) [inline src/stylesheets/layouts/_header.scss]: delete comment

File: src/stylesheets/layouts/_header.scss [hunk 3]
```diff
@@ -0,0 +1,136 @@
+@use "../abstracts/" as *;
+
+.header-container {
```
Comment (Hoa0304) [inline src/stylesheets/layouts/_header.scss]: Rename for ease of use

File: src/stylesheets/layouts/_header.scss [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+        height: pxToRem(95);
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding: 0 pxToRem(120);
+
+        // logo
+        &__logo {
+            .header-top__logo-item {
+                display: flex;
```
Comment (Hoa0304) [inline src/stylesheets/layouts/_header.scss:17]: Change code according to BEM in SCSS

File: src/stylesheets/layouts/_header.scss [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+                align-items: center;
+                column-gap: 14px;
+
+                .logo {
```
Comment (Hoa0304) [inline src/stylesheets/layouts/_header.scss]: ?

File: src/stylesheets/layouts/_header.scss [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+                span {
+                    color: $color-main;
+                    font-weight: 600;
+                }
+            }
+        }
+
+        // nav header top
+        &__nav-container {
```
Comment (Hoa0304) [inline src/stylesheets/layouts/_header.scss]: same

File: src/stylesheets/layouts/_header.scss [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+                font-size: 1rem;
+            }
+        }
+    }
+
+    .header-title {
+        width: 100%;
+        height: 758px;
+        // set bg header
+        background-image: linear-gradient(0, white, rgba(204, 204, 216, 0.868)), url(./../assets/img/backgroud.png);
```
Comment (Hoa0304) [inline src/stylesheets/layouts/_header.scss:55]: bg should be placed in the public file

**Review by Hoa0304 — COMMENTED** [review]
Please change all tag names in the scss file

**Comment by nhatanh19** [comment]
> Please change all tag names in the scss file

Thank you for the reminder, I will fix it

File: src/stylesheets/layouts/_header.scss (hunk 7 above)
Comment (nhatanh19) [reply src/stylesheets/layouts/_header.scss:55]: > bg should be placed in the public file

Sorry I don't really understand this, can you explain more clearly?

**Pull request closed without merging**